package/wpewebkit/wpewebkit.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-WPEWEBKIT_VERSION = 2.22.0
+WPEWEBKIT_VERSION = 2.22.1
```

Comment:
We have [just released 2.22.2 yesterday](https://wpewebkit.org/release/wpe-2.22.2.html). Could you be so kind to update the PR to pick the new release? Thanks!

Reply:
Sure.  I am working on it. I will be busy for next week Hopefully i will try to test and submit today

Reply:
Done